_posts/2015-02-20-gocumber.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  "Gocumber"
+date:   2015-02-20 09:00:00
+categories: golang
+comments: true
+author: "Phil Trimble"
+---
+
+This month Sittercity Tech has released a new project on github: [gocumber!](https://github.com/sittercity/gocumber "gocumber")
```

Comment:
Addressed in https://github.com/sittercity/techblog/commit/d178bc4c14000652f3970f5f98673ce0cc59b49a